cardano-api/src/Cardano/Api/TextView.hs

```diff
@@ -51,11 +50,12 @@ newtype TextViewDescription
 -- It also contains a \"title\" field which is free-form, and could be used
 -- to indicate the role or purpose to a reader.
 --
-data TextView = TextView
-  { tvType :: !TextViewType
-  , tvDescription :: !TextViewDescription
-  , tvRawCBOR :: !ByteString
-  } deriving (Eq, Show)
+data TextView
+  = TextView { tvType        :: !TextViewType
```

Comment:
Would be nice to fix:
* unneeded new line for the `= TextView`
* insane vertical alignment.

Reply:
I've disable record formatting altogether.